dapp-test/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Dapp Test</title>
+    <style>
+      * {
+        padding: 0;
+        margin: 0;
+      }
+      canvas {
+        background: #050d4f;
+        display: block;
+        margin: 0 auto;
+      }
+    </style>
+  </head>
+  <body>
+    <script type='text/javascript' src="./massa-wallet-provider.js"></script>
+    <script type='text/javascript' src="./massa-wallet-provider-content-script.js"></script>
```

Comment:
[content script](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts) is intended to be loaded by an extension, not in an application. the title Dapp test makes me thing that this is a website application, not an extension.